src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/dto/ServiceClientDto.java

```diff
@@ -31,10 +31,10 @@
 import java.time.OffsetDateTime;
 
 /**
- * DTO for Service and ServiceClient access rights
+ * DTO for ServiceClient data
  */
 @Data
-public class AccessRightHolderDto {
+public class ServiceClientDto {
```

Comment:
Now that I read the code, I find it pretty confusing that this dto has localGroupDescription and memberName (which is not used in ServiceClientConverter, though) but no global group description.

Reply:
No changes done.